mathesar_ui/src/component-library/cancel-or-proceed-button-pair/CancelOrProceedButtonPair.svelte

```diff
@@ -2,21 +2,15 @@
   import Button from '@mathesar-component-library-dir/button/Button.svelte';
   import SpinnerButton from '@mathesar-component-library-dir/spinner-button/SpinnerButton.svelte';
   import Icon from '@mathesar-component-library-dir/icon/Icon.svelte';
-  import {
-    iconCancel,
-    iconProceed,
-  } from '@mathesar-component-library-dir/common/icons';
   import type { Size } from '@mathesar-component-library-dir/commonTypes';
   import type { ButtonDetails } from './CancelOrProceedButtonPairTypes';
 
   const cancelButtonDefaults: ButtonDetails = {
     label: 'Cancel',
-    icon: iconCancel,
   };
 
   const proceedButtonDefaults: ButtonDetails = {
     label: 'Proceed',
-    icon: iconProceed,
   };
```

Comment:
Can you add these icons back in as defaults? I know that Figma doesn't show icons for these buttons, but that's okay. I think it's probably just because Ghislaine didn't want to take the time to put icons in all the buttons.

Reply:
Done 4cd6563